### PR TITLE
All seqs dropped

### PR DESCRIPTION
**Pull request by wasade**
Test and report if all seqs filtered out, stage for micro release.

### CODE REVIEW

**Comment by wasade** [comment]
@eldeveloper @antgonza @rnaer @mortonjt anyone able to review?

**Comment by RNAer** [comment]
👍 , though I only looked at the changes.

**Comment by antgonza** [comment]
👍  note that I don't have merge rights.

**Comment by wasade** [comment]
Thanks! I had a conversation with @jairideout this morning, and it sounds like the recommendation is to not cut a micro release for this. In addition, the strategy I used for the micro version differs from the recommended approach, notably, branching off the tag instead of just off master -- which in hindsight makes much more sense. I can't do it immediately, but I'll roll back the commits effecting the version changes, and just keep this fix in `master` for the time being.

**Comment by wasade** [comment]
@jairideout I committed a fix for #22 on this PR as well. In my opinion, #22 warrants a microrelease as it does impact utilization on known datasets, and can lead to difficult to diagnose bugs. Perhaps we should discuss this on the dev call?

**Comment by jairideout** [comment]
We've noticed the integer ID problem in other plugins too. Let's discuss a strategy for fixing these on the dev call.

**Comment by gregcaporaso** [comment]
I agree that it's a good topic for the dev call, but I'll weigh in now since I can't join this week. 

While this is an important issue I don't think it's critical to do a patch release since it's a known bug in a few plugins in this release. We're planning to address this across the plugins for the March release (which is only about 4-5 weeks away), so this would then be released with those fixes. (We'll also most likely have a new integration test that covers this case at that time too.) Since we're aware of the issue now, we can help users if it comes up in the meantime. 

That said, if you do want to address it with a patch release, I'm not opposed to that. I think @jairideout shared a list of the steps to get that ready.

**Comment by jairideout** [comment]
@wasade are you wanting this included in the 2017.4 release?

Note that the version strings were updated upstream by @gregcaporaso recently so that part of the diff needs to match upstream.

**Comment by wasade** [comment]
Just pull upstream? It improves interpretation of a plausible error so it would be good to have but not critical

**Comment by jairideout** [comment]
> Just pull upstream? It improves interpretation of a plausible error so it would be good to have but not critical

I don't understand your comment, can you rephrase/expand? This PR diff declares incorrect versions; we want to use the ones that are currently in `master`. The only changes we'd want to merge here are your changes in `q2_quality_filter/_filter.py` and `q2_quality_filter/test/test_filter.py`. Can you update this PR so that I can merge it?

**Comment by wasade** [comment]
I'm not sure what exactly you'd like changed. Can you just issue the commit? It'll likely be faster as, otherwise, I worry that I'll make some changes which will not be sufficient yielding multiple iterations to resolve.

**Comment by jairideout** [comment]
Replaced by #23.
